experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -238,37 +237,42 @@ describe('fetch', () => {
       new tracing.SimpleSpanProcessor(dummySpanExporter)
     );
 
+    // endSpan is called after the whole response body is read
+    // this process is scheduled at the same time the fetch promise is resolved
+    // due to this we can't rely on getData resolution to know that the span has ended
+    let resolveEndSpan: (value: unknown) => void;
+    const spanEnded = new Promise(r => resolveEndSpan = r);
+    const readSpy = sinon.spy(window.ReadableStreamDefaultReader.prototype, 'read');
+    const endSpanStub: sinon.SinonStub<any> = sinon.stub(FetchInstrumentation.prototype, <any>'_endSpan')
```

Comment:
Nit: `as` typecasting should be preferred over bracket typecasting.

```suggestion
    const endSpanStub: sinon.SinonStub<any> = sinon.stub(FetchInstrumentation.prototype, '_endSpan' as any)
```